src/util/Options.js

```diff
@@ -253,18 +222,6 @@ class Options extends null {
   static get defaultMakeCacheSettings() {
     return {
       MessageManager: 200,
-      ChannelManager: {
-        sweepInterval: 3600,
-        sweepFilter: require('./Util').archivedThreadSweepFilter(),
-      },
-      GuildChannelManager: {
-        sweepInterval: 3600,
-        sweepFilter: require('./Util').archivedThreadSweepFilter(),
-      },
-      ThreadManager: {
-        sweepInterval: 3600,
-        sweepFilter: require('./Util').archivedThreadSweepFilter(),
-      },
```

Comment:
Just to be sure, are these included already?

Reply:
What do you mean by that?

Reply:
Do the new sweeper options also autosweep archived threads like the old sweepers?

Reply:
Yes, its in 1 key now though (nice benefit)